scripts/globals/settings.lua

```diff
@@ -83,6 +83,7 @@ HARVESTING_RATE         = 0.50; --% chance to recieve an item from haresting.  S
 EXCAVATION_RATE         = 0.50; --% chance to recieve an item from excavation.  Set between 0 and 1.
 LOGGING_RATE            = 0.50; --% chance to recieve an item from logging.  Set between 0 and 1.
 MINING_RATE             = 0.50; --% chance to recieve an item from mining.  Set between 0 and 1.
+STRANGE_APPARATUS_RATE  = 0.50; --% chance to recieve an item other than a crystal cluster from a strange apparatus.  Set between 0 and 1.

```

Comment:
With doctor status on retail I have never got a cluster at all, ever. Pretty sure it outright removes them from possible results. Since most people are going to calc their password and pretty much always have that status, the setting to adjust the cluster rate will not often figure in and be mostly unused.

As for other items [ffxiclopedia](http://wiki.ffxiclopedia.org/wiki/Strange_Apparatus) likely a pretty accurate list of what you can get if you haven't already referenced that.